src/Components/animation/AnimationEditor.js

```diff
@@ -121,10 +114,7 @@ class AnimationEditor extends React.Component {
    */
   _positionPixelToPercent = (x, y) => {
     /* Here we assume that there is no view at the left or the bottom of the editor */
-    return [
-      (x - this.state.dLeft) / this.state.animationWidth,
-      (y - this.state.animationTopMargin) / this.state.animationHeight,
```

Comment:
It looks like it is taken into account in the setting of `dTop`, so we don't need it anymore